include/sdf/system_util.hh

```diff
@@ -26,20 +26,22 @@
  */
 
 #if defined _WIN32 || defined __CYGWIN__
-  #ifdef BUILDING_DLL
+  #ifdef BUILDING_SDFORMAT_SHARED
     #ifdef __GNUC__
       #define SDFORMAT_VISIBLE __attribute__ ((dllexport))
     #else
       #define SDFORMAT_VISIBLE __declspec(dllexport)
     #endif
```

Comment:
mingw supports `__declspec(dllexport)` and `__declspec(dllimport)`
See http://www.mingw.org/wiki/exporting_a_class_from_a_dll_for_multiple_instantiations

A special case for `__GNUC__` is not needed, imho.

Reply:
Makes sense